### ISSUE TITLE
Sidebar css broken with rutorrent 4.3.0

### ISSUE THREAD

**Issue by 3z3qu13l** [QuickBox/club-QuickBox#9]
Hi,
the sidebar is kinda broken with rutorrent 4.3.0

Here is the layout:
![image](https://github.com/QuickBox/club-QuickBox/assets/34472031/01321d6e-ac31-4779-ba27-b3e73a67ad64)

**Comment by Nirzak** [comment]
Yeah I agree. this is most likely may be because of rutorrent's recent material design lay out that's why

**Comment by w4hnsinn** [comment]
Anyone with css skills able to fix this? 

Alternatively, are there any other decent themes? Do not like the built-in ones at all.
